lib/qless/job.rb

```diff
@@ -110,6 +111,18 @@ def self.build(client, klass, attributes = {})
       new(client, attributes)
     end
 
+    # Converts a hash of job options (as returned by job.to_hash) into the array
+    # format the qless api expects.
+    def self.build_opts_array(opts)
+      result = []
```

Comment:
agreed a little unsure of it atm. had to stop work mid way through the branch =) picking up the work again today.